tests/modinfo/tests/tests.rs

```diff
@@ -0,0 +1,27 @@
+use std::env;
+use std::fs;
+use std::path::Path;
+use std::process::Command;
+
+use kernel_module_testlib::with_kernel_module;
+
+#[test]
+fn test_modinfo() {
+    let module = env::var("KERNEL_MODULE").unwrap();
+
+    let description = Command::new("modinfo").arg("-d").arg(module).output().unwrap();
+    assert!(description.status.success());
+    assert_eq!(std::str::from_utf8(&description.stdout).unwrap().trim(), "Empty module for testing modinfo");
```

Comment:
Probably ought to test for author and license while we're here.

Reply:
License is tested by the taint check, but sure, worth doing explicitly.